src/posts/lucky_1.4.1.cr

```diff
@@ -0,0 +1,182 @@
+    since you will need Crystal on any machine you're using anyway. You can now generate a new Lucky app on Windows, and
+    bootstrap the app with `crystal script/setup.cr`.
+
+    ### Separate Read/Write databases
+
+    Avram now supports using separate read and write databases for queries. By default, this will continue to use your
+    single specified database.
+
+    ```crystal
+    abstract class BaseModel < Avram::Model
+      def self.read_database : Avram::Database.class
+        ReaderDB
+      end
+
+      def self.write_database : Avram::Database.class
+        WriterDB
+      end
+    end
+
+    # Uses the ReaderDB
+    user = UserQuery.find(1)
+
+    # Uses the WriterDB
+    SaveUser.update!(user, last_active_at: Time.utc)
+    ```
+
+    Currently this will only change the connection between doing queries and saving records through Save or Delete operations.
+    Handling tasks such as fallbacks, or retries based on latency from reading and writing are not supported.
+
+    ### Table Locking
+
+    In general, a table lock is a way to lock your table while you perform operations on it preventing other connections
+    from altering that table during that time. In Postgres, however, there's 8 different types of locking that can
+    allow for a lot more customization and handling when it comes to locking your table.
+
+    The `AppDatabase.with_lock_on` method takes the model whose table you will be locking, and the locking `mode`.
+    Then within the block, all queries made will happen within the transaction.
+
+    ```crystal
+    AppDatabase.with_lock_on(User, mode: :row_exclusive) do
+      user = UserQuery.new.id(1).for_update.first
+      SaveUser.update!(user, name: "New Name")
+    end
+    ```
+
+    Avram supports the following lock modes:
+
+    ```
+    ACCESS_SHARE
+    ROW_SHARE
+    ROW_EXCLUSIVE
+    SHARE_UPDATE_EXCLUSIVE
+    SHARE
+    SHARE_ROW_EXCLUSIVE
+    EXCLUSIVE
+    ACCESS_EXCLUSIVE
+    ```
+
+    ### Type-safe ENV methods
+
+    LuckyEnv got a few updates as well. When you use LuckyEnv, it will generate methods for each of your
+    environment variables and automatically type-cast them depending on the value. The supported types are
+    `String`, `Bool`, `Int32`, and `Float64`.
+
+    ```
+    CACHE_ENABLED=true
+    PORT=3000
+    HOST="localhost"
+    EXCHANGE_RATE=3.14
+    ```
+
+    With this `.env` file loaded in, you now have access to these methods:
+
+    ```crystal
+    LuckyEnv.cache_enabled? #=> true
+    LuckyEnv.port #=> 3000
+    LuckyEnv.host #=> "localhost"
+    LuckyEnv.exchange_rate #=> 3.14
+    ```
+
+    ### State of LuckyFlow
+
+    If you're not familiar, LuckyFlow is a shard that ships with new browser based Lucky apps that
+    helps with doing Acceptance testing by spinning up a browser and walking through a user's flow
+    using your site.
+
+    As you can see, we went from v1.4.0 right to v1.4.1. This was due to some bugs we found in LuckyFlow
```

Comment:
You're also tee-ing up every paragraph here (e.g. "As you can see") and then saying what you came to say. Not a big deal, but you might try starting with the statement and adding the additional language only if it's necessary.

One way could be:

```suggestion
    We released LuckyFlow v1.4.1 and skipped v1.4.0. This was due to some bugs we found in LuckyFlow
```

Reply:
Good catch. I do that a lot :sweat_smile: